### PR TITLE
Update README with command line for MSBuild using private packages fo…

### PR DESCRIPTION
**Pull request by aziel**
Added a comment with the line for building it while Nuget can't resolve between Razor 2 and Razor 3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ To get the source code of our SDKs and samples via **git** just type:
 
 Run "C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe .\Microsoft.WindowsAzure.Mobile.Build.msbuild"
 
+NOTE:
+If you are having trouble building with the above line, you can force MsBuild to use a private packages directory by setting NugetPackageSource appropriately: 
+"C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe /p:NugetPackageSource="C:\git\azure-mobile-services-quickstarts\backend\dotnet\private-packages;https://www.nuget.org/api/v2/" Microsoft.WindowsAzure.Mobile.Build.msbuild"
```
Comment (phvannor) [inline README.md:27]: This is assuming the install locations of MSBuild (granted already being done earlier) and where the repo was cloned to.  Can the path be relative when passed into MSBuild?

Comment (aziel) [inline README.md:27]: I tried a relative path ( backend\dotnet...   and .\backend\dotnet... ) and they didn't work.

Comment (fabiocav) [inline README.md:27]: You should be able to use "%cd%\backend\donet\private-packages".

Comment (fabiocav) [inline README.md:27]: We might just want to add a build batch/cmd file to make this a little simpler.

Comment (brettsam) [inline README.md:27]: You can actually add it to the .msbuild file, where you can probably take advantage of MSBuild properties to use the current directory. Then it's built-in automatically and you don't need to specify it.
`<DefaultNugetPackageSource>https://www.nuget.org/api/v2/</DefaultNugetPackageSource>`

**Comment by phvannor** [comment]
@brettsam says this is no longer needed, closing PR.

**Pull request closed without merging**